src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -59,7 +59,11 @@ def register_cli_argument(self, *args, **kwargs):
             register_cli_argument(*args, **kwargs)
         else:
             from azure.cli.core.commands.parameters import ignore_type
-            kwargs = {'arg_type': ignore_type}
+            if 'arg_type' in kwargs:
+                kwargs = {'arg_type': ignore_type}
+            else:
+                kwargs = {}
+                args = tuple([args[0], args[1], ignore_type])
```

Comment:
Because the type can be specified either by kwarg (arg_type=blah) or as an arg.  Previously it only worked when arg_type was specified as a kwarg, so this change makes it work for both. (basically I ran into a bug as I was working and this fixed it :smile:)